beaconApp/src/main/java/com/pulselink/beacon/ui/ThreadScreen.kt

```
@@ -202,9 +264,47 @@ private fun MessageBubble(message: SmsMessageItem, theme: ThemePalette) {
+    )
+    resolved.forEach { resolveInfo ->
+        context.grantUriPermission(
+            resolveInfo.activityInfo.packageName,
+            uri,
+            Intent.FLAG_GRANT_READ_URI_PERMISSION
+        )
+    }
+
+    try {
+        context.startActivity(Intent.createChooser(intent, "Send attachment via SMS/MMS"))
+    } catch (e: ActivityNotFoundException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)